src/wallet.ts

```diff
@@ -339,11 +349,13 @@ export class Wallet {
     })
 
     // @ts-ignore: method does not exist on Contract type
-    await contract.nft_batch_transfer(
-      { token_ids: tokenIds },
-      MAX_GAS,
-      ONE_YOCTO
-    )
+    await contract.nft_batch_transfer({
+      meta: 'nft_batch_transfer',
+      callbackUrl: options?.callbackUrl,
+      args: { token_ids: tokenIds },
+      gas: gas,
+      amount: ONE_YOCTO,
```

Comment:
So we willfully ignore anything that a user of mintbase-js would supply as amount? Contract panics on anything else than 1 yocto, but I consider it better not to silently override user input.

Reply:
Yes only for the ones we know the contract panics.